.pre-commit-config.yaml

```diff
@@ -103,6 +103,13 @@ repos:
             |\.\.\ code-block\ ::
             |\.\.\ ipython\ ::
         types_or: [python, cython, rst]
+    -   id: cython-casting
+        name: Cython Casting
+        language: pygrep
+        entry: |
+            # Cython casting is of the form `<type>obj` as opposed to `<type> obj`
+            |[a-zA-Z0-9*]>\s
+        types: [cython]
```

Comment:
`pxi.in` isn't recognised as `cython` by [identify](https://github.com/pre-commit/identify), does the following work?

```suggestion
        files: (\.pyx|\.pxi.in)$
```

Reply:
I tried that, but it shows the following message, don't know if it worked or not as it showed the same message when I ran unwanted patterns with the cython check their. While, it gave error for .rst files when ran with ci-check on PR.

<img width="1030" alt="Screenshot 2021-08-02 at 8 51 19 PM" src="https://user-images.githubusercontent.com/46242526/127885070-14358a8b-6d6f-452b-99a6-3bbac71bdd41.png">

Reply:
Should I make a commit anyway?

Reply:
ah, you should run `pre-commit run cython-casting --all-files`, else it'll only run against files you've staged

Reply:
Thank you!! I ran that and found that my hook is not working for the previous test.pyx or test.pix.in example, it works when I move the logic to unwanted hooks but it doesn't work with the new hook

Reply:
Ok with the quotes it works now